_posts/2023-02-22-#25038.md

```diff
@@ -7,18 +7,258 @@ authors: [glozow]
+- Ephemeral outputs may be used to enable [Eltoo](https://bitcoinops.org/en/topics/eltoo/), which also
+[hinges](https://bitcoinops.org/en/newsletters/2022/05/18/#using-transaction-introspection-to-prevent-rbf-pinning) on the resolution of these RBF limitations.
+- As v3 and package RBF would allow counterparties to replace each others' commitment transactions, just a single anchor
+  output per commitment transaction could be sufficient. This may eventually allow the removal of
+[CPFP carve out](https://github.com/bitcoin/bitcoin/blob/master/doc/policy/mempool-limits.md#cpfp-carve-out).
+
 ## Questions
-1. Did you review the PR? [Concept ACK, approach ACK, tested ACK, or NACK](https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#peer-review)? What was your review approach?
--->
 
+1. Did you review the PR? [Concept ACK, approach ACK, tested ACK, or NACK](https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#peer-review)?
+What was your review approach?
+
+### Concept
+
+1. What is the difference between a transaction that "explicitly" signals BIP125 and one that "inherits" BIP125
+   signaling? Why is it impossible for a v3 transaction to be one but not the other?
+
+1. Why should there be a rule to check whether a replacement transation is more incentive compatible to mine?
+
+1. Is min(package feerate, individual feerate) an accurate incentive compatibility score for v3 transactions? What
+   about in general? (See `CheckMinerScores()` [implementation](https://github.com/bitcoin-core-review-club/bitcoin/commit/58e811ecb1e1977422ecda2af62460e8efc057be)).
+
+1. Imagine that we *really* need a transaction txA to confirm immediately, but a malicious counterparty has broadcast a conflicting
+   transaction txB which is 2000vB, paying 10,000sat in fees (5sat/vB). Let's say we are ok with either txA or txB
+confirming, as long as it's in the next block, and it's guaranteed (just pretend, ok?) that anything above 20sat/vB will
+be mined immediately, while anything below 20sat/vB will not.  What fee can we put on our transaction to *guarantee*
+that either txA or txB will reach a package feerate of 20sat/vB? (Hint: txA needs to replace txB and any descendants it
+may have.  What is the maximum fee the counterparty can attach to descendants of txB without bumping it to 20sat/vB?)
+
+1. If all mining nodes on the network are running with this v3 policy, while none of the non-mining nodes are, what happens?
+
+1. If 10% of nodes on the network are running with this v3 policy, while we haven't upgraded our node yet, what happens?
+   What about 99% of nodes?
+
+1. If we were running a business that accepted unconfirmed transactions as payments, what changes should we make so that our wallet correctly tracks v3 transactions?
+
+1. Who might be interested in using v3 transactions? Who probably wouldn't be interested? If there are some wallets/uers/applications that use v3 while others don't, is there a fingerprinting issue?
```

Comment:
Would add privacy - not sure everyone is familiar with the term
```suggestion
1. Who might be interested in using v3 transactions? Who probably wouldn't be interested? If there are some wallets/uers/applications that use v3 while others don't, is there a (privacy) fingerprinting issue?
```

Reply:
Expanded what I mean by fingerprinting